test/router-luatest/router_test.lua

```diff
@@ -545,3 +545,45 @@ g.test_enable_disable = function(g)
     -- we don't want this server to interfere with subsequent tests
     vtest.drop_instance(g, router)
 end
+
+g.test_simultaneous_cfg = function()
+    local router = vtest.router_new(g, 'router_1')
+
+    router:exec(function(cfg)
+        ivshard.router.internal.errinj.ERRINJ_CFG_DELAY = true
+        rawset(_G, 'fiber_cfg_static', ifiber.new(ivshard.router.cfg, cfg))
+        rawset(_G, 'fiber_cfg_new', ifiber.new(ivshard.router.new,
+                                               'new_router', cfg))
+        _G.fiber_cfg_static:set_joinable(true)
+        _G.fiber_cfg_new:set_joinable(true)
+    end, {global_cfg})
+
+    local routers_cfg = function()
```

Comment:
I would propose to declare functions rather like that:
```Lua
local function routers_cfg()
```
instead of
```Lua
local routers_cfg = function()
```
See motivation in old comment in the storage commit.

Reply:
Fixed)